# msr2g_enrU

## 2/10/17

Serpent run yielded k\_eff of 1.03

![msr2g_enrU](msr2g_enrU_mesh1.png)

# msr2g_part_U_single_cell

## 2/10/17

Serpent run yielded k$_{eff}$ of 1.44

![msr2g_part_U_single_cell](msr2g_part_U_single_cell_mesh1.png)

# msr2g_part_U_full_core

## 2/13/17

Serpent run yielded k$_{eff}$ of .79216 with
- D ~ 55 cm
- H = 162.56 cm
- vacuum BCs

![msr2g_part_U_full_core](msr2g_part_U_full_core_mesh1_small_radius.png)

Dimensions based on buckling size given on pg. 39, MSRE Design and Operations, part iii, nuclear analysis

- k$_{eff}$ = 1.07
- D = 74.93 cm
- H = 198.12 cm
- vacuum BCs
- graphite as special moderator material

![msr2g_part_U_full_core_bigger](msr2g_part_U_full_core_mesh1.png)

- k$_{eff}$ = 1.0545
- D = 74.93 cm
- H = 198.12 cm
- vacuum BCs
- graphite as regular doppler broadened material

![msr2g_part_U_full_core_bigger](msr2g_part_U_full_core_mesh1_doppler_graphite.png))

### 1144 K fuel

- k$_{eff}$ = 1.04149
- Fuel temperature coefficient of reactivity = -3.08e-5 [$\frac{\delta k}{k} / ^{\circ}F$]

In [4]:
k_nom = 1.0545
k_f_1144 = 1.04149

fuel_reactivity = (k_f_1144 - k_nom) / k_nom / 400
print(fuel_reactivity)

-3.0844001896633395e-05


### 1144 K fuel and graphite

- k_eff = 1.02315

In [8]:
k_f_g_1144 = 1.02315
total_reactivity = (k_f_g_1144 - k_nom) / k_nom / 400
print(total_reactivity)
graph_reactivity = total_reactivity - fuel_reactivity
print(graph_reactivity)

-7.43243243243243e-05
-4.348032242769091e-05


In [10]:
from numpy import abs
print(abs(total_reactivity + 6.96e-5) / 6.96e-5 * 100)
print(abs(fuel_reactivity + 3.28e-5) / 3.28e-5 * 100)
print(abs(graph_reactivity + 3.68e-5) / 3.68e-5 * 100)

6.78782230506
5.96340885173
18.1530500752


# Summary

## Serpent Fuel C calculation

- $\alpha_f = -3.08\cdot 10^{-5} [\frac{\delta k}{k} / ^{\circ}F]$
- $\alpha_g = -4.34\cdot 10^{-5} [\frac{\delta k}{k} / ^{\circ}F]$
- $\alpha_{tot} = -7.43\cdot 10^{-5} [\frac{\delta k}{k} / ^{\circ}F]$

## MSRE report

- $\alpha_f = -3.28\cdot 10^{-5} [\frac{\delta k}{k} / ^{\circ}F]$
- $\alpha_g = -3.68\cdot 10^{-5} [\frac{\delta k}{k} / ^{\circ}F]$
- $\alpha_{tot} = -6.96\cdot 10^{-5} [\frac{\delta k}{k} / ^{\circ}F]$

## % error
- $\alpha_f \rightarrow 5.96%$
- $\alpha_g \rightarrow 18.15%$
- $\alpha_{tot} \rightarrow 6.79%$

In [3]:
from numpy import exp

alpha_fuel_faeh = 1.18e-4
alpha_fuel_kel = 1.8 * alpha_fuel_faeh
alpha_graph_faeh = 1.0e-5
alpha_graph_kel = 1.8 * alpha_graph_faeh
rho0_fuel = 2.146
rho0_graph = 1.86
T0 = 922
rho1144_fuel = rho0_fuel * exp(-alpha_fuel_kel * (1144 - T0))
rho1144_graph = rho0_graph * exp(-alpha_graph_kel * (1144 - T0))
print(rho1144_fuel)
print(rho1144_graph)

2.04715872548
1.85258227049


# msre_homogeneous

## 2/13/17

Serpent run yielded k$_{eff}$ of 1.03

![msre_homogeneous](msre_homogeneous_mesh1.png)

# General Notes

## 2/13/17
- pg. 39, MSRE Design and Operations, part iii, nuclear analysis: For one-region model of reactivity effects of temperature, geometric buckling used was that of a cylinder, **59** in. in diameter by **78** in. high. Three conditions:
    - salt and graphite at 1200 F
    - salt at 1600 F, graphite at 1200 F
    - salt and graphite at 1600 F
- Temperature Reactivity coefficients both negative for fuel and graphite
- Most definite reactor geometry and material descriptions: pages 18 & 19 of MSRE Design and Op., part iii
- On those pages is a 2D R-Z 20-region model; within each region the material composition is considered to be **uniform**
- Critical concentrations of Uranium were calculated for three different salt compositions